apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -70,11 +70,14 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
 
       <h1 className="mt-12 text-4xl font-semibold">Documents</h1>
 
-      <div className="mt-8 flex flex-wrap gap-x-4 gap-y-6">
-        <Tabs defaultValue={shouldDefaultToPending ? InternalDocumentStatus.PENDING : status}>
+      <div className="mt-8 flex flex-wrap gap-x-4 gap-y-6 ">
+        <Tabs
+          className="overflow-auto"
```

Comment:
This should probably just be `overflow-x-auto` since we don’t want any y axis scrolling 😄